docs_src/getting-started/quick-start/index.md

```diff
@@ -1,227 +1,255 @@
-This guide will get EdgeX up and running on your development machine in as little as 5 minutes. We will will skip over lengthy descriptions for now, you can read up on those later. The goal here is to get you a working IoT Edge stack, from device to cloud, as simply as possible.
-
+This guide will get EdgeX up and running on your development machine in as little as 5 minutes. We will skip over lengthy descriptions for now, you can read up on those later. The goal here is to get you a working IoT Edge stack, from device to cloud, as simply as possible.
 
 ## Setup
-
-
 The fastest way to start running EdgeX is by using our pre-built Docker images. To use them you'll need to install the following:
 
 * Docker <https://docs.docker.com/install/>
 * Docker Compose <https://docs.docker.com/compose/install/>
 
-
-
 ## Running EdgeX
+Once you have Docker and Docker Compose installed, you need the file for downloading and running the EdgeX Foundry docker containers. Download the latest [`docker-compose` file](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis-no-secty.yml) and save this as `docker-compose.yml` in your local directory. This file contains everything you need to deploy EdgeX with docker.  
 
+!!! Note
+    The above is the Geneva release of EdgeX for use on **Intel x86** platforms.  If you are running on ARM, use this [docker-compose file instead](https://github.com/edgexfoundry/developer-scripts/blob/master/releases/geneva/compose-files/docker-compose-geneva-redis-no-secty-arm64.yml).
 
-Once you have Docker and Docker Compose installed, you need the configuration file for downloading and running the EdgeX Foundry docker containers. Download the latest `docker-compose` file here <https://github.com/edgexfoundry/developer-scripts/raw/master/releases/fuji/compose-files/docker-compose-fuji.yml> and save this as `docker-compose.yml` in your local directory. This file contains everything you need to deploy EdgeX with docker.
-
-First, use this command to download the EdgeX Foundry Docker images from Docker Hub:
+First, use this command to download the EdgeX Foundry Docker images from Docker Hub.  From a terminal window, change directory to your local directory containing the docker-compose.yml and execute the following Docker Compose command:
 ```
 docker-compose pull
 ```
-Then start up all of the EdgeX Foundry microservices:
+This fetches all the referenced EdgeX containers.  Then start up all of the EdgeX Foundry microservices:
 ```
 docker-compose up -d
 ```
-Finally, verify that the EdgeX containers have all been started:
+The -d option on docker-compose up is to start the containers in detached mode which allows the containers to run independently from the terminal window.  Finally, verify that the EdgeX containers have started:
 ```
 docker-compose ps 
```

Comment:
I dont think its incumbent upon our docs or our community to explain basic docker commands --  i think we forgo these descriptions and just use the command above.

Reply:
removed